src/components/ReviewMigration.tsx

```diff
@@ -56,7 +60,7 @@ function ReviewMigration({
               {commify(
                 formatBNToString(migrationAmount || BigNumber.from("0"), 18, 2),
               )}{" "}
-              saddleUSD
+              {isBTCMigration ? "saddleBTC" : "saddleUSD"}
```

Comment:
Why can't we just display the pool name as it is? Or get the pools lpToken name? Those sound more correct, esp since we may have an eth pool that gets migrated

Reply:
Sounds good to me

Reply:
Nice find btw